protocol/app/testdata/default_genesis_state.json

```diff
@@ -351,6 +351,16 @@
       "min_num_votes_per_sample": 15
     }
   },
+  "marketmap": {
+    "last_updated": "0",
+    "market_map": {
+      "markets": {}
+    },
+    "params": {
+      "admin": "dydx10d07y265gmmuvt4z0w9aw880jnsr700jnmapky",
+      "market_authorities": ["dydx10d07y265gmmuvt4z0w9aw880jnsr700jnmapky"]
```

Comment:
nit: Comment on which address this is? (gov?)

Reply:
This is the gov address - Left a comment and noted that we will need to add the skip market map authority as well

Reply:
yes this is the gov address.  Unfortunately cant add a comment to json